web/virtual_lab/views.py

```
@@ -67,3 +75,62 @@
+    "javascript": "js",
+    "c": "c",
+    "cpp": "cpp",
+}
+
+
+def code_editor_view(request):
+    return render(request, "virtual_lab/code_editor/code_editor.html")
+
+
+@require_POST
+def evaluate_code(request):
```

Ruff (0.11.9)

Missing return type annotation for public function `evaluate_code`

(ANN201)